Added scratchpad.py to get the Fellows started.

treuille opened the pull request:
### Summary

Added a file called `Insight_Project_Framework/Insight_Project_Framework/scratchpad.py` to the base repo used by the students which helps them get started with streamlit.

treuille committed:
Added scratchpad.py to get the fellows started.

mrubash1 reviewed (commented):
Hey Adrien, 

I would love to add this. Mind doing 3 quick things before we merge in.

- Move this file to the Insight_Project_Framework subfolder (used to be the src folder)
- Add a requirements.txt file with the streamlit library to the main area
- Add a quick install instructions in the readme for pip installing streamlit

Thanks!

treuille committed:
Updated installation instructions and added requirements.txt.

treuille commented:
@mrubash1 : I think I addressed your concerns. In particular I made the following changes:
- `scratchpad.py` is in `Insight_Project_Framework`. (Actually, it already was. Did I miss something here?)
- Added a `requirements.txt` to the top-level folder.
- Added installation instructions to the *Requisites* section of the top-level `readme.md` file.

mrubash1 reviewed (approved):
+1
